src/app/content/components/Toolbar.tsx

```diff
@@ -0,0 +1,164 @@
+import { maxNavWidth } from '../../components/NavBar';
+import { contentFont, textRegularLineHeight, textRegularSize, textRegularStyle } from '../../components/Typography';
+import theme from '../../theme';
+import { assertString } from '../../utils';
+import SidebarControl from './SidebarControl';
+
+const iconColor = '#5E6062';
+
+export const toolbarDesktopHeight = 5;
+export const toolbarMobileHeight = 4;
+
+export const iconStyles = css`
+  height: 1.6rem;
+  width: 1.6rem;
+  margin-right: 0.5rem;
+  color: ${iconColor};
+`;
+
+// tslint:disable-next-line:variable-name
+const SearchIcon = styled(Search)`
+  ${iconStyles};
+`;
+
+// tslint:disable-next-line:variable-name
+const PrintIcon = styled(Print)`
+  ${iconStyles};
+`;
+
+// tslint:disable-next-line:variable-name
+const ListIcon = styled(ListOl)`
+  ${iconStyles};
+`;
+
+// tslint:disable-next-line:variable-name
+const ToCButton = styled.h3`
+  font-family: ${contentFont};
+  ${textRegularSize};
+  color: ${iconColor};
+  margin: 0;
+  padding: 0;
+`;
+
+// tslint:disable-next-line:variable-name
+const ToCButtonWrapper = styled(SidebarControl)`
+  border: none;
+  padding: 0;
+  margin: 0;
+  overflow: visible;
+  background: none;
+  display: flex;
+  align-items: center;
+`;
+
+// tslint:disable-next-line:variable-name
+const TopBar = styled.div`
+  height: ${toolbarDesktopHeight}rem;
+  max-width: ${maxNavWidth}rem;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  overflow: visible;
+
+  ${theme.breakpoints.mobile(css`
+    height: ${toolbarMobileHeight}rem;
+  `)}
+`;
+
+// tslint:disable-next-line:variable-name
+const SearchInputWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  margin-right: 4rem;
+  border-bottom: solid 0.1rem ${iconColor};
+
+  ${theme.breakpoints.mobile(css`
+    border: none;
+    margin-right: 1rem;
+  `)}
+`;
+
+// tslint:disable-next-line:variable-name
+const SearchInput = styled.input`
+  ${textRegularStyle}
+  color: ${theme.color.text.default};
+  height: ${textRegularLineHeight}rem;
+  border: none;
+  outline: none;
+
+  ::placeholder {
+    color: ${iconColor};
+  }
+  ${theme.breakpoints.mobile(css`
+    display: none;
+  `)}
+`;
+
+// tslint:disable-next-line:variable-name
+const PrintOptWrapper = styled.div`
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+`;
+
+// tslint:disable-next-line:variable-name
+const PrintOptions = styled.h3`
+  ${textRegularStyle};
+  color: ${iconColor};
+  margin: 0;
+
+  ${theme.breakpoints.mobile(css`
+    display: none;
+  `)}
+`;
+
+// tslint:disable-next-line:variable-name
+const SearchPrintWrapper = styled.div`
+  text-align: right;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+// tslint:disable-next-line:variable-name
+const BarWrapper = styled.div`
+  width: 100%;
+  padding: 0 ${theme.padding.page.desktop}rem;
+  box-shadow: 0 0.2rem 0.2rem 0 rgba(0,0,0,0.14);
```

Comment:
It seems like the box shadow is included in several places. Could this be extracted out as a function? (maybe that would not help with maintainability)

Reply:
it is used twice i think in the navs, but i'm not sure it would really help to extract it since its one line and the different usages could easily want different shadows